obspy/imaging/waveform.py

```diff
@@ -500,7 +500,7 @@ def plot_day(self, *args, **kwargs):
         if "min_magnitude" in events:
             try:
                 from obspy.clients.fdsn import Client
-                c = Client("NERIES")
+                c = Client("EMSC")
```

Comment:
I guess this should become a parameter in the long run. Also `ISC` might be the correct resource but it likely returns a metric ton of events which might be too much for a plot.